### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigrationInprogress_Migration Cancel:Not able to navigate inside Cancel migration dialog box using keyboard.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15861]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter 
4. Tab till "Data migration in progress" and press enter.
5. Tab till ongoing migration under database table header and press enter.
6. Tab till cancel migration button and press enter.

### Actual:
Not able to navigate inside Cancel migration dialog box using keyboard.

### Expected:
Cancel migration dialog box should be accessible using keyboard .

### User Impact:
Keyboard only users will not be able to access the cancel migration dialog box.

**Comment by aasimkhan30** [comment]
A toast pops up and it does not take focus. The ask is to make it have focus.

**Comment by prabhudevu** [comment]
I have verified the issue in the below build 

Version: 1.43.0-insider (user setup)
Commit: afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed.

**GithubTags:**#Closed;#Regressed:03-29-23;

![cancel migration](https://user-images.githubusercontent.com/98821792/228513572-cb58a5ac-c07c-48bb-95ab-470d5ef6efa7.gif)
